docs/views/reference/filters.jade

```diff
@@ -4,8 +4,11 @@ block documentation
   h2 Filters
 
   p.
-    Filters let you use other languages within a jade template.
+    Filters let you use other languages within a jade template. 
     They take a block of plain text as an input.
+  p.
+    All #[a(href='//www.npmjs.com/browse/keyword/jstransformer' target='_blank') JSTransformers] can be used as jade filters. 
```

Comment:
Please use "https://" url here.  Protocol relative urls are an anti-pattern.  Please also remove `target='_blank'`.  If users want to open in a new tab they can do that by holding down CMD.